docs/developer/checkout/api-guide.mdx

```diff
@@ -430,3 +433,248 @@ mutation {
+          "streetAddress1": "11 W 53rd St, New York, NY",
+          "streetAddress2": "",
+          "countryArea": "NY",
+          "country": {
+            "country": "United States of America",
+            "code": "US",
+            "__typename": "CountryDisplay"
+          },
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "__typename": "Address"
+        }
+      }
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 2,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+</TwoColumn>
+
+<TwoColumn>
+<div>
+## Update Delivery Method
+
+Use [`checkoutDeliveryMethodUpdate`](/api-reference/checkout/mutations/checkout-delivery-method-update.mdx) to pass the delivery method returned by the `checkout` query.
+
+Notice that the return type in `checkout.deliveryMethod` can be either `Warehouse` or `ShippingMethod`, so we need to use [union type](https://graphql.org/learn/schema/#union-types).
+
+Learn more about [delivery methods](/developer/checkout/address#selecting-the-delivery-method.mdx).
+
+</div>
+<Tabs>
+<TabItem value={"Mutation"}>
+```graphql
+mutation {
+  checkoutDeliveryMethodUpdate(
+    id: "Q2hlY2tvdXQ6MDk1MmNiNWUtMzZkYi00YTQ5LThhN2MtZjAyNGE2M2Y1NzNj"
+    deliveryMethodId: "U2hpcHBpbmdNZXRob2Q6MTU="
+  ) {
+    errors {
+      message
+      field
+    }
+    checkout {
+      deliveryMethod{
+        ...on Warehouse{
+          id
+          name
+        }
+        ... on ShippingMethod{
+          id
+          name
+        }
+      }
+    }
+    __typename
+  }
+}
+```
+</TabItem>
+<TabItem value={"Response"}>
+```json
+{
+  "data": {
+    "checkoutDeliveryMethodUpdate": {
+      "errors": [],
+      "checkout": {
+        "deliveryMethod": {
+          "id": "U2hpcHBpbmdNZXRob2Q6MTU=",
+          "name": "FedEx"
+        }
+      },
+      "__typename": "CheckoutDeliveryMethodUpdate"
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 0,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+</TwoColumn>
+
+<TwoColumn>
+<div>
+## Completing checkout
+Checkout can be completed if all [requirements are satisfied](/developer/checkout/lifecycle#completing-checkout).
+
+When the checkout is fully paid, the webhook [`CHECKOUT_FULLY_PAID`](api-reference/webhooks/enums/webhook-event-type-async-enum.mdx#webhookeventtypeasyncenumcheckout_fully_paid) will be triggered.
+
+</div>
+<Tabs>
+<TabItem value={"GraphQL"}>
+```graphql {2-4}
+mutation {
+  checkoutComplete(
+    id: "Q2hlY2tvdXQ6ZTEzZDFjOTItOWJkNi00ODViLTgyMDctZTNhM2I5NjVkZTQw"
+  ) {
+    order {
+      id
+      status
+    }
+    errors {
+      field
+      message
+    }
+  }
+}
+```
+</TabItem>
+<TabItem value={"Result"}>
+```json
+{
+  "data": {
+    "checkoutComplete": {
+      "order": {
+        "id": "T3JkZXI6MjU=",
+        "status": "UNFULFILLED"
+      },
+      "errors": []
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+</TwoColumn>
```

Comment:
Not sure if this is scope of this PR, but I think it would be worth to also explain mutations like `checkoutPromoCodeAdd`, `checkoutCustomerAttach/Detach` etc

Reply:
Added extra block 
<img width="574" alt="image" src="https://github.com/user-attachments/assets/708d51a4-0f20-4851-9b66-4e029d1d8b44">